react/src/components/ProductsTree.js

```diff
@@ -0,0 +1,121 @@
+// import { Component } from 'react';
```

Comment:
If we're adding a new file that's entirely commented out, consider adding a comment at the top explaining under what conditions we would comment it back in? Or if you plan on re-commenting soon, no problem and no need to do anything